src/pages/hackathon/radar.tsx

```diff
@@ -12,10 +12,10 @@ import { Meta } from '@/layouts/Meta';
 import { statsDataQuery, trackDataQuery } from '@/queries/hackathon';
 import { RadarLogo } from '@/svg/radar-logo';
 
-const MotionBox = motion(Box);
-const MotionFlex = motion(Flex);
-const MotionText = motion(Text);
-const MotionButton = motion(Button);
+const MotionBox = motion(Box as any);
```

Comment:
why?

Reply:
packages updated, this broke.
minor version of these packages was updated, (like eslint broke), same reason, things broke, typescript errors